src/main/java/com/targomo/client/Constants.java

```diff
@@ -55,6 +55,7 @@
     public static final String X                                                        = "x";
     public static final String CALLBACK                                                 = "callback";
     public static final String SRID                                                     = "srid";
+    public static final String FORCE_RIGHT_HAND_ORIENTATION                             = "forceRightHandOrientation";
```

Comment:
does it perhaps make sense to have an enum? 
Because i can totally forsee that later there will be a "forceLeftHandOrientation"